tpl/collections/collections_test.go

```diff
@@ -582,6 +587,34 @@ func TestQuerify(t *testing.T) {
 	}
 }
 
+func BenchmarkQuerify(b *testing.B) {
+	ns := New(&deps.Deps{})
+	params := []interface{}{"a", "b", "c", "d", "f", " &"}
+
+	b.ReportAllocs()
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		_, err := ns.Querify(params...)
+		if err != nil {
+			b.Fatal(err)
+		}
+	}
+}
+
+func BenchmarkQuerifySlice(b *testing.B) {
+	ns := New(&deps.Deps{})
+	params := []string{"a", "b", "c", "d", "f", " &"}
+
+	b.ReportAllocs()
```

Comment:
We dont't need/want ReportAllocs in our benchmarks (you can toggle that on/off when running these.